orc8r/cloud/deploy/bare-metal-ansible/roles/magma/orc8r/files/orc8r-configs.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
I would avoid duplicating this. The metricsd config can change in future, it might necessitate us making same change across several places going forward. Let's use the same config file if possible.

Reply:
This file is templated also in Terraform and I'm not sharing code across the terraform deploy logic. I would prefer if the orc8r helm chart could generate it

Reply:
I'm switching back to a symlink